src/panels/lovelace/editor/config-elements/hui-stack-card-editor.ts

```diff
@@ -219,7 +219,7 @@ export class HuiStackCardEditor
   }
 
   private _getKey(cards: LovelaceCardConfig[], index: number): string {
-    const key = `${cards[index].type}${index}${cards.length}`;
+    const key = `${index}${cards.length}`;
```

Comment:
```suggestion
    const key = `${index}-${cards.length}`;
```
This should have some separator. Otherwise 1-11 and 11-1 would generate the same key. Probably isn't a problem but it just makes me nervous to concat 2 numbers and just get a new number.